stylus/base/helpers.styl

```diff
@@ -62,27 +62,35 @@ for $alignment, $text-align in $alignments
   +mobile()
     .has-text-{$alignment}-mobile
       text-align: $text-align !important
+
```

Comment:
Please remove these trailing newlines, for better block jumping, and keep the style from our source project: https://github.com/jgthms/bulma/blob/master/sass/base/helpers.sass#L56-L83